app/static/src/app/components/options/EditParamSettings.vue

```diff
@@ -161,9 +175,26 @@ export default defineComponent({
         const centralValue = computed(() => store.state.run.parameterValues[settingsInternal.parameterToVary!]);
 
         const paramValues = computed(() => store.state.run.parameterValues);
-        const batchParsResult = computed(() => generateBatchPars(store.state, settingsInternal, paramValues.value));
-        const batchPars = computed(() => batchParsResult.value.batchPars);
-        const batchParsError = computed(() => batchParsResult.value.error);
+        const batchParsResult = computed(() => {
+            if (settingsInternal.variationType === SensitivityVariationType.Custom) {
+                return null;
+            }
+            return generateBatchPars(store.state, settingsInternal, paramValues.value);
+        });
+        const batchPars = computed(() => batchParsResult.value?.batchPars);
+        const batchParsError = computed(() => {
+            if (settingsInternal.variationType === SensitivityVariationType.Custom) {
+                // Minimum of two custom values
+                return settingsInternal.customValues.length < 2
+                    ? { error: "Invalid settings", detail: "Must include at least 2 traces in the batch" } : null;
+            }
+            return batchParsResult.value?.error;
+        });
+        const updateUserValues = (newValues: number[]) => {
+            // sort and remove duplicates
```

Comment:
when I entered a duplciate it ended up with a red underline and was not included but no error displayed

Reply:
I also wonder about deferring sort until save? it's a bit alarming atm

Reply:
The red underline seems to be a bit of a feature of the third party tags input, I think it can happen in the advanced settings too, I'll see if I can get rid of it, and defer sorting